audit-connector/src/main/java/com/sequenceiq/cloudbreak/audit/model/AuditEventName.java

```diff
@@ -1,5 +1,25 @@
 package com.sequenceiq.cloudbreak.audit.model;
 
 public enum AuditEventName {
-    DATAHUB_CLUSTER_CREATION
+    CREATION_DATAHUB_CLUSTER,
+    DELETION_DATAHUB_CLUSTER,
+    STOPPAGE_DATAHUB_CLUSTER,
+    RESIZING_DATAHUB_CLUSTER,
+    MANUAL_REPAIR_DATAHUB_CLUSTER,
+    SYNC_DATAHUB_CLUSTER,
+    RETRY_DATAHUB_CLUSTER,
+    INSTANCE_DELETION_DATAHUB_CLUSTER,
+    MAINTENANCE_DATAHUB_CLUSTER,
+    STARTING_DATAHUB_CLUSTER,
+
+    CREATION_DATALAKE_CLUSTER,
+    DELETION_DATALAKE_CLUSTER,
+    STOPPAGE_DATALAKE_CLUSTER,
+    RESIZING_DATALAKE_CLUSTER,
+    MANUAL_REPAIR_DATALAKE_CLUSTER,
+    SYNC_DATALAKE_CLUSTER,
+    RETRY_DATALAKE_CLUSTER,
+    INSTANCE_DELETION_DATALAKE_CLUSTER,
+    MAINTENANCE_DATALAKE_CLUSTER,
+    STARTING_DATALAKE_CLUSTER
```

Comment:
Please rename the corresponding constants to the following, as it was recommended in the event names sheet:

CREATE_DATALAKE_CLUSTER
DELETE_DATALAKE_CLUSTER
STOP_DATALAKE_CLUSTER
RESIZE_DATALAKE_CLUSTER
DELETE_DATALAKE_CLUSTER_INSTANCE
MAINTAIN_DATALAKE_CLUSTER
START_DATALAKE_CLUSTER

Reply:
done